accessibility-oscilloscope/configs/qemu_defconfig

```diff
@@ -1,5 +1,5 @@
 BR2_arm=y
-BR2_TOOLCHAIN_BUILDROOT_WCHAR=y
```

Comment:
the breaking change here comes from needing glibc for numpy.

Reply:
Ah rip numpy

Reply:
Is WCHAR support is included in glibc already or something? I don't see it in menuconfig with glibc enabled so I assume that's how it works.